Orange/widgets/data/owtranspose.py

```diff
@@ -72,7 +73,7 @@ def __init__(self):
         self.apply_button.button.setAutoDefault(False)
 
         self.info.set_output_summary(self.info.NoInput)
-        self.info.set_input_summary(self.info.NoInput)
+        self.info.set_input_summary(self.info.NoOutput)
```

Comment:
`set_input_summary` should use `self.info.NoInput`.
`set_output_summary ` should use `self.info. NoOutput `.